content/study/posts/liberia_custom.yml

```diff
@@ -0,0 +1,96 @@
+title: Liberia National Electrification Analysis (NEA) (custom visualization)
+bbox: [[-11.438, 4.355],[-7.539, 8.541]]
+zoomExtent: [ 3, 10 ]
+mapConfig: ./liberia_custom-mb.json
+country: Liberia
+study:
+  consultant: NRECA
+  period: 2017-2018
+  content: ./liberia_custom.md
+charts:
+  - name: Modelled Electricity demand (GWh/year)
+    type: donut
+    data:
+      - name: Residential
+        value: 90
+      - name: Social (HF, EF, public)
+        value: 60
+      - name: Commercial
+        value: 60
+      - name: Productive
+        value: 60
+      - name: Other
+        value: 30
+  - name: Total additional capacity (MW)
+    type: donut
+    data:
+      - name: Grid
+        value: 1500
+      - name: Mini-grid
+        value: 150
+      - name: Stand-alone
+        value: 600
+  - name: Total lengh of T&D
+    type: number
+    datum: 
+      value: 60
+      unit: km
+  - name: Mini-grids
+    type: number
+    datum: 
+      value: 65
+      unit:
+  - name: Stand-alone systems
+    type: number
+    datum: 
+      value: 3000
+      unit:
+platform:
+  title: Liberia National Electrification Analysis (NEA)
+  url: https://liberianea.com/index.php/view/
+layers:
+  - id: transmission_lines_1
+    name: Liberia Electricity Transmission Network (2009)
+    category: input
+    mbLayer: transmission_lines_1
+    info: Data for medium and high voltage transmission lines in Liberia (2009). The data were compiled for the AICD study led by the World Bank. A variety of sources were consulted, including regional power pool documents and maps from World Bank project documents.
+    source:
+      name: energydata.info
+      url: https://energydata.info/dataset/liberia-electricity-transmission-network
```

Comment:
```suggestion
      url: https://energydata.info/dataset/liberia-electricity-transmission-network
    displayData:
      - label: From
        valueProp: FROM_NM
      - label: To
        valueProp: TO_NM
      - label: Voltage
        valueProp: VOLTAGE_KV|suffix:kv
```

This is an example of how a popover would be customized for the `transmission_lines_1` layer.
This would result in the following:
![image](https://user-images.githubusercontent.com/1090606/121885424-0dd07900-cd0c-11eb-927e-f8469edefe7b.png)

All the popover customization options can be found at: https://github.com/developmentseed/aep/tree/main/docs#customizing-popup-data

Reply:
Thanks @danielfdsilva, I have now editted the pop up window for the population dataset and seems to work fine.